CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+Use device apikey if exists in getEffectiveApiKey
```

Comment:
IOTA-UL agent would need a similar modification?

Reply:
This is the PR for iota-ul https://github.com/telefonicaid/iotagent-ul/pull/384